bcipy/helpers/task.py

```diff
@@ -370,186 +373,159 @@ def get_key_press(
                    timing_info: list,
-                   eeg_data: np.array,
-                   fs: int, k: int, mode: str,
+                   eeg_data: np.ndarray,
+                   fs: int,
+                   trials_per_inquiry: Optional[int] = None,
                    offset: float = 0,
-                   channel_map: tuple = DEFAULT_CHANNEL_MAP,
-                   trial_length: float = 0.5) -> tuple:
-    """Trial Reshaper.
-
-    Trail reshaper is used to reshape trials, based on trial target info (target, non-target),
-        timing information (sec), eeg_data, sampling rate (fs), down-sampling rate (k),
-        mode of operation (ex. 'calibration'),
-        offset (any calculated or hypothesized offsets in timings),
-        channel map (which channels to include in reshaping) and
-        trial length (length of reshaped trials in secs)
-
-    In all modes > calibration, we assume the timing info is given in samples not seconds.
-
-    :return (reshaped_trials, labels, num_of_inquiries, trials_per_inq)
-
-        reshaped_trials is a 3 dimensional np array where the first dimension
-            is the channel, second dimension the trial/letter, and third
-            dimension is time samples. So at 300hz and a presentation time of
-            0.2 seconds per letter, the third dimension would have about 60
-            samples.
-            If this were a json structure it might look like:
-                {
-                    'ch1': {
-                        'B' : [-647.123, -93.124, ...],
-                        'T' : [...],
-                        ...
-                    },
-                    'ch2': {
-                        'B' : [39.60, -224.124, ...],
-                        'T' : [...],
-                        ...
-                    },
-                    ...
-                }
+                   channel_map: List[int] = DEFAULT_CHANNEL_MAP,
+                   trial_length: float = 0.5,
+                   targets_included: Set[str] = set(["target", "nontarget"]),
+                   targets_excluded: Set[str] = set([])) -> Tuple[np.ndarray, np.ndarray]:
+    """Extract trial data and labels.
+
+    Args:
+        trial_target_info (list): labels each trial as "target", "non-target", "first_pres_target", etc
+        timing_info (list): Timestamp of each event in seconds
+        eeg_data (np.ndarray): shape (channels, samples) preprocessed EEG data
+        fs (int): sample rate of preprocessed EEG data
+        trials_per_inquiry (int, optional): unused, kept here for consistent interface with `inquiry_reshaper`
+        offset (float, optional): Any calculated or hypothesized offsets in timings. Defaults to 0.
+        channel_map (tuple, optional): Describes which channels to include or discard. Defaults to DEFAULT_CHANNEL_MAP.
+        trial_length (float, optional): [description]. Defaults to 0.5.
+        targets_included (Set[str]): target labels to include
```

Comment:
labels would be better now that I think of it more. Not necessary to adjust here unless you agree passionately, there a few places we use target including `trial_target_info` where we mean label.

Reply:
I do agree with this, and perhaps `trial_labels` would be a reasonable name to use instead of `trial_target_info` - however as I started making this change, I realized this occurs in a TON of places.

Let's take care of the renaming in a follow-up?

Reply:
(And I'll try to use clear naming within these reshaper functions themselves for the time being)

Reply:
https://www.pivotaltracker.com/story/show/178308208